pkg/porter/build_integration_test.go

```diff
@@ -1,29 +1,36 @@
+// +build integration
```

Comment:
I made this an integration test because it hits the filesystem and relies on build having been run first.